Routing/Loader/Reader/RestActionReader.php

```diff
@@ -352,16 +352,16 @@ private function getCondition(\ReflectionMethod $method, RouteAnnotation $annota
         $condition = $annotation->getCondition();
 
         if (!empty($this->versions)) {
-            $versionCondition = "request.attributes.get('version') == (";
+            $versionCondition = "request.attributes.get('version') in [";
             $first = true;
             foreach ($this->versions as $version) {
```

Comment:
Ok now I undertand :+1:  Thx for the explanation. The old solution with the `or` is totally wrong indeed. I also fixed that problem in my other PR though:

https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1438